Packs/CommonPlaybooks/Playbooks/playbook-Block_Domain_-_Generic_v2.yml

```diff
@@ -0,0 +1,336 @@
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": -240,
+          "y": 310
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: 56106ad2-8509-4c5b-89a6-8c8cc10223c5
+    type: title
+    task:
+      id: 56106ad2-8509-4c5b-89a6-8c8cc10223c5
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ""
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 680,
+          "y": 530
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "6":
+    id: "6"
+    taskid: 8db4ff8c-a46d-4be6-8669-a157e9ecea30
+    type: playbook
+    task:
+      id: 8db4ff8c-a46d-4be6-8669-a157e9ecea30
+      version: -1
+      name: Block Domain - Trend Micro Apex One
+      playbookName: Block Domain - Trend Micro Apex One
+      type: playbook
+      iscommand: false
+      brand: ""
+      description: ""
+    nexttasks:
+      "#none#":
+        - "5"
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 1140,
+          "y": 310
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "7":
+    id: "7"
+    taskid: 54dcc2d1-497f-4c3c-800a-897d125a2dbd
+    type: playbook
+    task:
+      id: 54dcc2d1-497f-4c3c-800a-897d125a2dbd
+      version: -1
+      name: Block Domain - Proofpoint Threat Response
+      playbookName: Block Domain - Proofpoint Threat Response
+      type: playbook
+      iscommand: false
+      brand: ""
+      description: ""
+    nexttasks:
+      "#none#":
+        - "5"
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 1600,
+          "y": 310
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "9":
+    id: "9"
+    taskid: 71a65711-416b-4a49-89b9-9ec5251c5936
+    type: playbook
+    task:
+      id: 71a65711-416b-4a49-89b9-9ec5251c5936
+      version: -1
+      name: Block Domain - Cisco Stealthwatch
+      playbookName: Block Domain - Cisco Stealthwatch
+      type: playbook
+      iscommand: false
+      brand: ""
+      description: ""
+    nexttasks:
+      "#none#":
+        - "5"
+    scriptarguments:
+      Domain:
+        simple: ${inputs.Domain}
```

Comment:
Please use the UI filters&transformers and pick the input.
![image](https://user-images.githubusercontent.com/28757135/183605305-89cb8ebb-ada7-4477-bbc5-8e8729bfc1aa.png)